paasta_tools/cli/cmds/logs.py

```diff
@@ -167,7 +168,24 @@ def prefix(input_string, component):
     return "%s: %s" % (LOG_COMPONENTS[component]['color'](component), input_string)
 
 
-def paasta_log_line_passes_filter(line, levels, service, components, clusters):
+def check_timestamp_in_range(timestamp, start_time, end_time):
+    """A convenience function to check if a timestamp is within the given start and end times,
+    returns true if start_time or end_time is None
```

Comment:
Because `start_time` and `end_time` are sent as kwargs to all the `filter_*` functions, this way I can pass them straight into this function without having if statements everywhere.